src/client/portforwarder.go

```diff
@@ -70,12 +70,16 @@ func NewPortForwarder(context *config.Context, namespace string) (*PortForwarder
 
 // Run starts the port forwarder. Returns after initialization is begun with
 // the locally bound port and any initialization errors.
-func (f *PortForwarder) Run(appName string, localPort, remotePort uint16) (uint16, error) {
+func (f *PortForwarder) Run(appName string, localPort, remotePort uint16, selectors ...string) (uint16, error) {
```

Comment:
The JupyterHub chart uses a slightly different label scheme, so we need to be able to filter by `component: proxy` as well.